examples/projections/cyl/cyl_universal_transverse_mercator.py

```diff
@@ -2,19 +2,34 @@
 Universal Transverse Mercator
 =============================
 
-A particular subset of the transverse Mercator is the Universal Transverse
-Mercator (UTM) which was adopted by the US Army for large-scale military maps.
-Here, the globe is divided into 60 zones between 84°S and 84°N, most of which
-are 6° (in longitude) wide. Each of these UTM zones have their unique central
-meridian. Furthermore, each zone is divided into latitude bands but these are
-not needed to specify the projection for most cases.
+A particular subset of the
+:doc:`transverse Merctor </projections/cyl/cyl_transverse_mercator>`
+is the Universal Transverse Mercator (UTM) which was adopted by the US Army
+for large-scale military maps. Here, the globe is divided into 60 zones
+between 84°S and 84°N, most of which are 6° (in longitude) wide.
+Each of these UTM zones have their unique central meridian.
+Furthermore, each zone is divided into latitude bands but these are
+not needed to specify the projection for most cases. See Figure
+:ref:`Universal Transverse Mercator <GMT_utm_zones>` for all zone designations.
+
+.. _GMT_utm_zones:
+
+.. figure::
+   https://docs.generic-mapping-tools.org/latest/_images/GMT_utm_zones.png
```

Comment:
Adding `#noqa: W505` should make the style check pass:
```suggestion
.. figure:: https://docs.generic-mapping-tools.org/latest/_images/GMT_utm_zones.png # noqa: W505 
```

Reply:
Thanks @seisman. Good to know for next time!